src/org/traccar/api/resource/NotificationResource.java

```diff
@@ -49,7 +49,7 @@
             userId = getUserId();
         }
         Context.getPermissionsManager().checkUser(getUserId(), userId);
-        return Context.getNotificationManager().getUserNotifications(userId);
+        return Context.getNotificationManager().getAllUserNotifications(userId);
```

Comment:
We can just use the list for the current user, but I guess we can keep it as well.